message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1416,6 +1416,16 @@
       <field type="float" name="cross_track_error" units="m">Deepstall cross track error in meters (only valid when in DEEPSTALL_STAGE_LAND)</field>
       <field type="uint8_t" name="stage" enum="DEEPSTALL_STAGE">Deepstall stage, see enum MAV_DEEPSTALL_STAGE</field>
     </message>
+    <!-- cellular network status -->
+    <message id="199" name="CELLULAR_STATUS">
+      <description>cellular network status</description>
+      <field type="char[5]" name="technology">current cellular technology: 3G, LTE, WCDMA, etc...</field>
```

Comment:
Should be an enumeration

Reply:
I use char[5] for 2 reasons:
1. There are many cellular technology and it is not easy to list all possible string reported by carrier and test it
2. Cellular technology is continuous advancing  (for example: 5G is coming). If we use an enumeration, we need to keep it update.

Reply:
Maybe both? Make an enum of a bunch of common ones, then one enum named "other" which uses the string. Otherwise the string is ignored.